com.ca.lsp.cobol/lsp-core-domain/src/test/java/com/broadcom/lsp/domain/cobol/databus/impl/DatabusConfigProvider.java

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright (c) 2019 Broadcom.
```

Comment:
if you will apply changes please change the copyright year as well

Reply:
👌